plugins/core/jetbrains-community/src/software/aws/toolkits/jetbrains/services/amazonq/QConstants.kt

```diff
@@ -7,3 +7,216 @@ object QConstants {
+        "cljc",
+        "cljs",
+        "cob",
+        "cbl",
+        "cpy",
+        "cobra",
+        "coffee",
+        "cfm",
+        "cfc",
+        "lisp",
+        "lsp",
+        "v",
+        "cr",
+        "d",
+        "dart",
+        "pas",
+        "dpr",
+        "dfm",
+        "e",
+        "ex",
+        "exs",
+        "elm",
+        "el",
+        "elc",
+        "erl",
+        "hrl",
+        "fs",
+        "fsx",
+        "fsi",
+        "flow",
+        "f",
+        "f90",
+        "f95",
+        "gd",
+        "go",
+        "groovy",
+        "gvy",
+        "hack",
+        "hh",
+        "hs",
+        "lhs",
+        "html",
+        "htm",
+        "css",
+        "hy",
+        "pro",
+        "idl",
+        "io",
+        "java",
+        "class",
+        "jar",
+        "js",
+        "mjs",
+        "cjs",
+        "jl",
+        "kt",
+        "kts",
+        "lean",
+        "b",
+        "xc",
+        "x",
+        "xi",
+        "lgt",
+        "logtalk",
+        "lua",
+        "mpl",
+        "mv",
+        "m",
+        "nb",
+        "ma",
+        "m",
+        "m",
+        "mu",
+        "n",
+        "nim",
+        "nix",
+        "m",
+        "h",
+        "ml",
+        "mli",
+        "oz",
+        "pir",
+        "pasm",
+        "pas",
+        "pp",
+        "pl",
+        "pm",
+        "perl",
+        "php",
+        "phtml",
+        "pike",
+        "pmod",
+        "ps1",
+        "psm1",
+        "psd1",
+        "pro",
+        "pl",
+        "prolog",
+        "pp",
+        "purs",
+        "py",
+        "pyw",
+        "qs",
+        "r",
+        "rmd",
+        "rdata",
+        "rkt",
+        "rktl",
+        "raku",
+        "rakumod",
+        "rakutest",
+        "re",
+        "res",
+        "red",
+        "reds",
+        "rexx",
+        "rex",
+        "ring",
+        "roff",
+        "ron",
+        "rm",
+        "ms",
+        "rb",
+        "ruby",
+        "rbw",
+        "rs",
+        "rlib",
+        "sas",
+        "scala",
+        "sc",
+        "scm",
+        "ss",
+        "sb3",
+        "shen",
+        "st",
+        "sol",
+        "sql",
+        "sml",
+        "sig",
+        "scd",
+        "sc",
+        "swift",
+        "tcl",
+        "t",
+        "tu",
+        "oot",
+        "ts",
+        "tsx",
+        "vala",
+        "vapi",
+        "vba",
+        "bas",
+        "v",
+        "sv",
+        "vhd",
+        "vhdl",
+        "vb",
+        "vbx",
+        "zig",
+        // End of Stack Overflow list, add other relevant file extensions as needed starting next line
+        "json",
+        "md",
+        "scss",
+        "bat",
+        "cmake",
+        "f03",
+        "f08",
+        "f77",
+        "for",
+        "gql",
+        "graphql",
+        "gs",
+        "gsp",
+        "gsx",
+        "jsx",
+        "mak",
+        "makefile",
+        "sb",
+        "sb2",
+        "tf",
+        "xml",
+        "yaml",
```

Comment:
`.yml` is also valid for yaml

Reply:
+1

Reply:
as addressed previously, I will create a follow up PR to sync and re-evaluate the file extension constants